app/dashboard/_components/RiskAndOpps/Section.tsx

```diff
@@ -0,0 +1,35 @@
+import Card from '@/components/Card'
+
+// Assuming you have a type for the card data
+type CardData = {
+  title: string
+  description: string
+}
+
+type SectionProps = {
+  title: string
+  items: CardData[]
+}
+
+const Section = ({ title, items }) => {
+  return (
+    <section>
+      <h2 className='mb-4 ml-4 font-heading text-3xl font-bold md:text-4xl'>
+        {title}
+      </h2>
+      <div className='grid grid-cols-1 gap-4 md:grid-cols-2'>
+        {items.map((item) => (
+          <Card
```

Comment:
if you are also mapping out cards here, what is the RiskOppCard used for? Is it even used anywhere?